SplunkRumWorkspace/SplunkRum/SplunkRum/InstrumentationUtils.swift

```diff
@@ -72,6 +74,8 @@ class GlobalAttributesProcessor: SpanProcessor {
         span.setAttribute(key: "splunk.rumVersion", value: SplunkRumVersionString)
         span.setAttribute(key: "device.model.name", value: deviceModel)
         span.setAttribute(key: "os.version", value: UIDevice.current.systemVersion)
+        span.setAttribute(key: "latitude", value: locationData.latitude)
```

Comment:
If location data is not available, what is the value here for lat and lon?

Reply:
nill

Reply:
ok

Reply:
done